translator/totomlconfig/sampleConfig/prometheus_config_windows.json

```diff
@@ -63,8 +63,10 @@
             },
             {
               "label_matcher": "default",
+              "metric_namespace": "CustomizeMetricScrappingJobNamespace",
```

Comment:
What I'm saying is that there should be a unit test that remains untouched, so we know that this change doesn't break someone who has an older configuration but updates their agent. If we want to include a test for the job-specific metric namespace, I am saying there should be another configuration added that validates that specifically. We shouldn't be updating existing tests unless we have to - and needing to indicates a breaking change.

Reply:
For the parts of breaking changes, I agree. However, to validate the namespace, I hardly convinced that a unit test would be enough. It would need an integration test to bring up a mock server to validate these namespaces. However, would bring up a similar integration test's resource for a slightly different `metric_namespace` a good idea? Personally it's a no from me.